### PR TITLE
Update ko_kr.json

### PR DESCRIPTION
**Pull request by mindy15963**

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/ko_kr.json [hunk 1]
```diff
@@ -930,7 +931,7 @@
 	"option.securitycraft.keypad.sendMessage": "메시지: %s",
 	"option.securitycraft.keypad.sendMessage.description": "활성화 시 이 키패드는 메시지를 전송합니까(허용목록/거부목록 모듈 사용 시)?",
 	"option.securitycraft.keypad.signalLength": "신호 길이: %s",
-	"option.securitycraft.keypad.signalLength.description": "활성화 시 레드스톤 신호는 얼마나 걸립니까? (눈금 단위: 20틱 = 1초)",
+	"option.securitycraft.keypad.signalLength.description": "활성화 시 레드스톤 신호는 얼마나 길어야 합니까? (20틱 = 1초)",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/ko_kr.json:934]: Can you please explain this change? In your previous pull request, the line regarding the key panel's signal length was added like this: https://github.com/Geforce132/SecurityCraft/blob/fd15ee2/src/main/resources/assets/securitycraft/lang/ko_kr.json#L997 Why does this not apply here?

File: src/main/resources/assets/securitycraft/lang/ko_kr.json [hunk 2]
```diff
@@ -954,7 +955,7 @@
 	"option.securitycraft.keycard_reader.sendMessage": "메시지: %s",
 	"option.securitycraft.keycard_reader.sendMessage.description": "활성화 시 이 키카드 판독기가 메시지를 전송합니까(허용목록/거부목록 모듈 사용 시)?",
 	"option.securitycraft.keycard_reader.signalLength": "신호 길이: %s",
-	"option.securitycraft.keycard_reader.signalLength.description": "활성화 시 레드스톤 신호는 얼마나 걸립니까? (눈금 단위: 20틱 = 1초)",
+	"option.securitycraft.keycard_reader.signalLength.description": "활성화 시 레드스톤 신호는 얼마나 길어야 합니까? (20틱 = 1초)",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/ko_kr.json:958]: Same as my other comment.

File: src/main/resources/assets/securitycraft/lang/ko_kr.json (hunk 1 above)
Comment (mindy15963) [reply src/main/resources/assets/securitycraft/lang/ko_kr.json]: I just translated it by referring to the original. Why is there a problem?

Comment (bl4ckscor3) [reply src/main/resources/assets/securitycraft/lang/ko_kr.json]: My question was why `활성화 시 레드스톤 신호는 얼마나 걸립니까? (눈금 단위: 20틱 = 1초)` can be used for the key panel, but not for the keypad or keycard reader where you changed it back to just `활성화 시 레드스톤 신호는 얼마나 길어야 합니까? (20틱 = 1초)`. There is no difference between those two in the english translation, so I was wondering why there was in korean.

Comment (mindy15963) [reply src/main/resources/assets/securitycraft/lang/ko_kr.json]: There are same.

**Pull request merged**